packages/@aws-cdk/aws-ecs/test/ec2/ec2-service.test.ts

```diff
@@ -2621,6 +2621,37 @@ describe('ec2 service', () => {
       }).toThrow(/Cannot enable service discovery if a Cloudmap Namespace has not been created in the cluster./);
 
 
+    });
+
+    test('throws if cloudmap namespace is HTTP when enabling Service Discovery', () => {
```

Comment:
Failing tests should have names starting with "fails" per the design guidelines: https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#unit-tests

```suggestion
    test('fails to enable Service Discovery with HTTP defaultCloudmapNamespace', () => {
```